### PR TITLE
Docs 5865 ch required application updates hn

### PR DESCRIPTION
**Pull request by hannanelson**
Draft of topic ready for review. 

01/22/20: Updated PDF (with images):
[Updating the Runtime Version for CloudHub Apps _ MuleSoft Documentation.pdf](https://github.com/mulesoft/docs-runtime-manager/files/4100198/Updating.the.Runtime.Version.for.CloudHub.Apps._.MuleSoft.Documentation.pdf)

(old) Google doc (no images, but you can comment):
https://docs.google.com/document/d/144fFouVRaGw_stxkFlAr0byOUNyrRnRIolyeLoTnTMQ/edit?usp=sharing

### CODE REVIEW

**Comment by phuang2019** [comment]
Do we need a screenshot for app settings page with the notification banner? That banner will show an app specifc date for the auto update.

**Comment by hannanelson** [comment]
> Do we need a screenshot for app settings page with the notification banner? That banner will show an app specifc date for the auto update.

Good q for @evinaku. I didn't see this in the UI - is it there yet?

**Comment by phuang2019** [comment]
@hannanelson It is the second to last screenshot in my PR: https://github.com/mulesoft/cloudhub-ui/pull/1862
It is on the application settings page. But it will only show for app that is enabled in the database. You will have to coordinate with Nishant to enable the app you picked for you. Also someone keep overwriting my deployment in qax... We will have to redeploy before you take that screenshot.

**Comment by hannanelson** [comment]
@phuang2019 - 

OK. Let's see if @evinaku thinks it's necessary. Might not want a screenshot with an actual date - it might confuse people if their dates aren't the same - and will definitely make the docs outdated quickly.

**Comment by hannanelson** [comment]
@phuang2019, @evinaku - added screenshot for the app Settings page (with the banner).

**Review by phuang2019 — APPROVED** [review]

File: modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+Because these updates contain required security patches, you cannot roll back these updates once applied.
+
+
+== Update the App Manually 
+
+The *Update Required* tab indicates one or more apps require an update to Mule. You can schedule the update to occur at your convenience.
+
+Before updating apps in the Production environment, update them in the Sandbox environment and run required tests.
+
+To update the Mule version for an app:
```
Comment (nishantkakar) [inline modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc]: It's not updating the Mule Version of the app, it's more like going to the latest update of the Mule Version they are currently on
```suggestion
To update an app to the latest security patch:
```

**Review by nishantkakar — DISMISSED** [review]

File: modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+You can schedule and update your apps manually at your convenience or you can wait for MuleSoft to apply the updates automatically.
+
+MuleSoft updates CloudHub apps automatically, based on the environment, on this schedule:
+
+[%header,cols="2"]
+|===
+|Environment |App Update Schedule 
+|*Production*|Every three months
+|*Sandbox*|Once every month
```
Comment (nishantkakar) [inline modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc]: @vinayakjangam Please advise the updated schedule

**Review by vinayakjangam — APPROVED** [review]

**Review by phuang2019 — APPROVED** [review]

File: modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: cloudhub, runtime manager, arm, runtime version, mule, updates,
+
+image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
+image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", title="Hybrid Deployment"]
+image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
+
+To provide a safe cloud environment for applications and data, MuleSoft regularly scans apps deployed on CloudHub and identifies apps that require updates to the CloudHub runtimes.
```
Comment (evinaku) [inline modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc]: @hannanelson We are revisiting the strategy for these updates. So for now, I would like to change this section (up until the snapshot) to reflect the current status and update it again in a couple of week when we have a better schedule.

File: modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+[%header,cols="2"]
+|===
+|Environment |App Update Schedule 
+|*Production*|Every three months
+|*Sandbox*|Once every month
+|===
+
+When a new version is available and CloudHub application updates are required, the *Applications* page displays a banner and the *Update Required* tab lists the apps that require updates:
+
```
Comment (evinaku) [inline modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc:16]: @hannanelson Can the section above this be done away with for now and replaced with something like this:

"When you run an application on CloudHub, you get access to validated OS updates, security patches and bug fixes on a regular basis. You can keep you applications on CloudHub applications upto date by manually take the patch updates at your convenience or let the platform update these applications for you regularly.

When a new date patch version of the mule runtime your application is running is available the
Applications page displays a banner and the Update Required tab lists the apps that require
updates:"

We will revisit this once the strategy is crystalized.

File: modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc (hunk 3 above)
Comment (hannanelson) [reply modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc]: @evinaku - what is the current status?

**Review by evinaku — APPROVED** [review]

File: modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+endif::[]
+:keywords: cloudhub, runtime manager, arm, runtime version, mule, updates,
+
+image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies", title="CloudHub"]
+image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies", title="Hybrid Deployment"]
+image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
+
+When you deploy an application to CloudHub, you have access to the validated OS updates, security patches, and bug fixes that MuleSoft provides regularly.
+
+You can keep your CloudHub applications up to date by manually updating your apps at your convenience or you can let MuleSoft apply the updates automatically.
```
Comment (kbostrom) [inline modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc]:
```suggestion
You can keep your CloudHub applications up to date by manually updating your apps at your convenience, or you can let MuleSoft apply the updates automatically.
```

File: modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+When a new runtime version is available for your CloudHub application, the *Applications* page displays a banner and the *Update Required* tab lists the apps that require updates:
+
+.The screenshot shows (*1*) the banner and (*2*) the *Update Required* tab on the *Applications* page.
+image::cloudhub-app-update-req-page.png[Applications page with banner and Update Required tab]
+
+If no apps require updates, no banner appears and the *Update Required* tab is empty.
+
+== Update the App Manually 
+
+The *Update Required* tab indicates one or more apps require an update. You can schedule the update to occur at your convenience.
```
Comment (kbostrom) [inline modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc]:
```suggestion
The *Update Required* tab indicates that one or more apps require an update. You can schedule the update to occur at your convenience.
```

File: modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+
+Before updating apps in the Production environment, update them in the Sandbox environment and run required tests.
+
+To update an app to the latest security patch:
+
+. In Runtime Manager, in the menu on the left, click *Applications*.
```
Comment (kbostrom) [inline modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc]:
```suggestion
. In the Runtime Manager left menu, click *Applications*.
```
Tempted to say "In the Runtime Manager navigation menu, click...".

File: modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+. Click the *Update Required* tab.
+. Click the app status to display the details pane on the right.
```
Comment (kbostrom) [inline modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc]: How does one click the app status? Is there a label or icon or something in the UI?

File: modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+. Click the *Manage Application* button to display the *Settings* page for the app.
++
+If the app requires an update, *Update available* appears in the *Runtime version* field:
++
+.The screenshot shows (*1*) the *Update available* button and (*2*) the *Runtime version* menu on the *Settings* page.
+image::cloudhub-app-update-available-setting.png[Update available on the application Settings page]
+
+. Select the latest version from the *Runtime version* menu.
++
+Although the version number of the update might be the same as the installed version, the date distinguishes the update.
```
Comment (kbostrom) [inline modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc]:
```suggestion
Although the version number of the update might be the same as the installed version, a later date identifies the update.
```

File: modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+. If you want to see details about the version, click the *Read release notes* link.
+. Click the *Apply Changes* button to update the app. 
+
+CloudHub updates the app with the version you selected and displays a message indicating success or failure.
+
+If the app update was successful, the app no longer appears in the *Update Required* tab on the *Applications* page.
+
+If the app update was unsuccessful, the app remains in the *Update Required* tab.
```
Comment (kbostrom) [inline modules/ROOT/pages/cloudhub-app-runtime-version-updates.adoc:50]: Is there text following or a link topic that addresses how to troubleshoot this?

**Review by kbostrom — APPROVED** [review]
Voila!

**Pull request merged**